english/forest/forest.md

```diff
@@ -1,6 +1,6 @@
 An overwhelming pain echoes through your head. You feel the taste of dirt in
 your mouth. Your hands are moist and you can feel the impression of twigs and
-moss in your cheeks. You're in the forest.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.